package.json

```diff
@@ -122,7 +121,8 @@
     "typescript": "4.5.5"
   },
   "resolutions": {
-    "hawk": "7"
+    "hawk": "7",
+    "ember-cli-typescript": "link:."
```

Comment:
Huh, is this a standard Yarn pattern for self-resolution? I hadn't seen it before.

On reading the lock file changes, how does this impact consumers (if at all)?

Reply:
> Huh, is this a standard Yarn pattern for self-resolution?

I'm not sure if it's "standard" but it's definitely one I've used in a few places. In principle it could blow up on us in a future where we make big breaking changes here and our dependencies that rely on `ember-cli-typescript` aren't ready for that but are having it forced on them anyway. Since I don't think we're super likely to be making any changes like that, though... ¯\\\_(ツ)_/¯ 

> On reading the lock file changes, how does this impact consumers (if at all)?

I don't think it should impact consumers at all, since `resolutions` only apply in your project's own `package.json`, not your dependencies'.